### PR TITLE
SWTCH-968 Integrate new org/app/role definition creation into SB

### PR DESCRIPTION
**Pull request by jrhender**
Minor changes need to create and update Roles, Apps, Orgs using the new RoleDefinitionResolver smart contract:

- feat(new-role.component): use number version. As number was previously strings of the format "1.0.0", need to convert into a number as we are now using integers for role versions

Still in DRAFT as need to integrate updated iam-client-lib first

### CODE REVIEW

File: src/app/routes/applications/new-role/new-role.component.ts [hunk 1]
```diff
@@ -210,8 +210,8 @@ export class NewRoleComponent implements OnInit, AfterViewInit, OnDestroy {
       let parentNamespace = arrParentNamespace[1].substring(1);
 
       // Construct Version
-      let arrVersion = def.version.split('.');
-      let version = `${parseInt(arrVersion[0]) + 1}.0.0`;
+      const arrVersion = typeof(def.version) === 'string' ? parseInt(def.version.split('.')[0], 10) : def.version;
```
Comment (DawidGil-jlabs) [inline src/app/routes/applications/new-role/new-role.component.ts:213]: Could you extract this to a method? So those two lines disappear from here and in patching form you can assign result of this method to `version`.

Comment (jrhender) [reply src/app/routes/applications/new-role/new-role.component.ts]: @DawidGil-jlabs thanks, good tip. I updated the PR. Can you take a look and see if it is what you had in mind when you have time?

Comment (DawidGil-jlabs) [reply src/app/routes/applications/new-role/new-role.component.ts]: Yes, this is what i had in mind. Thanks!

**Review by DawidGil-jlabs — APPROVED** [review]
Lgtm

**Review by DawidGil-jlabs — APPROVED** [review]

**Pull request merged**